pay-queue/src/pay_queue/services/payment_reconciliations.py

```diff
@@ -247,9 +262,32 @@ def reconcile_payments(msg: Dict[str, any]):
 
     cas_settlement.processed_on = datetime.now()
     cas_settlement.save()
+    return has_errors, error_messages
+
+
+def _publish_mailer_error_events(file_name: str, minio_location: str, error_messages: List[Dict[str, any]]):
+    """Publish payment message to the mailer queue."""
+    payload = {
+        'fileName': file_name,
+        'errorMessages': error_messages,
+        'minioLocation': minio_location
+    }
+    try:
+        gcp_queue_publisher.publish_to_queue(
+            QueueMessage(
+                source=QueueSources.PAY_QUEUE.value,
+                message_type=QueueMessageTypes.EJV_FAILED.value,
```

Comment:
reused this one? you'll have to change it in the account-mailer or secrets.. probably should be noted in the description

Reply:
Also not necessary to use the queue messages.. technically you could just hit the `notify-api` directly

Reply:
Yeah you'll need to change account-mailer:

![image](https://github.com/bcgov/sbc-pay/assets/3484109/800cdfc2-df5a-47bb-b595-af451a87c12c)